deploy-manage/deploy/cloud-on-k8s/install-using-helm-chart.md

```diff
@@ -17,16 +20,14 @@ The minimum supported version of Helm is 3.2.0.
 ::::
 
 
-
 ## Cluster-wide (global) installation [k8s-install-helm-global]
```

Comment:
not urgent: would be good to have a little map to the sections on the page -"you can install cluster wide or restricted. you can also migrate an existing installation to helm"

consider also wrapping the install options in an h2 and making them h3s

Reply:
done! I've refined this page, hope you find it better now :)